src/common/entity/compute_state_name.ts

```diff
@@ -6,6 +6,9 @@ type CachedDisplayEntity = HassEntity & {
 };
 
 export default function computeStateName(stateObj: HassEntity) {
+  if (!stateObj) {
```

Comment:
We shouldn't do this, we should instead not call this method.

Reply:
I got an error cant call _entityDisplay from undefined, but can't reproduce it anymore...

Reply:
yeah that does happen when we pass undefined to it, we just shouldn't. 

To catch random exceptions, enable button ![image](https://user-images.githubusercontent.com/1444314/49735972-0cc8b780-fc89-11e8-8406-51dafdde6221.png) in the sources tab in Chrome.

Reply:
Yeah I know, just saying I don't know where undefined was passed... I'll just remove it then, and hope it will show up sometime with more info...